### ISSUE TITLE
NetworkAnimator Synchronised Parameters changes are not saved when editing in prefab mode

### ISSUE THREAD

**Issue by evbishop** [FirstGearGames/FishNet#544]
**General**
Unity version: 2021.3.18, 2022.3.13f1
Fish-Networking version: 3.11.5R.Pro and 3.11.7R.Pro
Discord link: https://discord.com/channels/424284635074134018/1172432283299483668

**Description**
If you turn off a Synchronized Parameter in NetworkAnimator while in prefab mode (the prefab was double clicked and is open), the change is gone once you exit prefab mode

**Replication**
1. Make a prefab with:
    1) an Animator that has an animator controller with some parameter;
    2) a NetworkAnimator.
2. Open the prefab in prefab mode.
3. Switch off the synchronized parameter in the NetworkAnimator.
4. Exit the prefab mode.
5. Select the prefab.

**Expected behavior**
The synchronized parameter in the NetworkAnimator keeps it's OFF (greyed out) value after it was toggled off in prefab mode.

**Comment by FirstGearGames** [comment]
It appears the object may not be getting dirtied. If I change a property on any other script after making changes and save it works. Should be a quick resolve, thanks!

**Comment by FirstGearGames** [comment]
Fixed in 3.11.8.
If you would like to apply the fix now open NetworkAnimatorEditor.cs.
Search for lines...
```cs
                                if (ignored)
                                    na.IgnoredParameters.Remove(parameterName);
                                else
                                    na.IgnoredParameters.Add(parameterName);
```
and right below those put `                   UnityEditor.EditorUtility.SetDirty(target);`
Your new code should look like this...
```cs
                                if (ignored)
                                    na.IgnoredParameters.Remove(parameterName);
                                else
                                    na.IgnoredParameters.Add(parameterName);
                                UnityEditor.EditorUtility.SetDirty(target);
```
